src/test/java/rest/gorestcoIn/utils/helpermethods/APICal.java

```diff
@@ -0,0 +1,35 @@
+package rest.gorestcoIn.utils.helpermethods;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+
+public class APICal {
+
+    public static Response post(String jsonBody, Header header) {
```

Comment:
header shouldn't be method parameter, it's not test data, which you generate/get from test layer, then pass here as parameter